src/protocol/protocol.ts

```diff
@@ -797,12 +799,12 @@ export namespace CDP {
   };
 
   export type SendCommandParams = {
-    cdpMethod: string;
+    cdpMethod: keyof ProtocolMapping.Commands;
```

Comment:
If you look at the top there is a comment that this file should not have dependencies. I have no context about why exactly.

Reply:
I believe it is because of the way Puppeteer uses Mapper. @OrKoN knows for sure.

Reply:
Type dependency should be alright because it is compiled away. The reason for no deps is that the file will be run by the browser so cannot import node_modules. E.g., put a note there so no one adds zod or smth like that in there. Thanks @Lightning00Blade for spotting.